infrastructure/azureInfrastructure.bicep

```diff
@@ -0,0 +1,180 @@
+param appName string = 'vote'
+param location string = 'Norway East'
+param staticWebAppLocation string = 'westeurope'
+
+/* 
+ Using a S1 plan to support development slots.
+ Also using a S1 plan to support more than 350 concurrent web socket connections.
+ If downgrading to a cheaper plan, such as B1, you could run into issues with the number of concurrent web socket connections under peak load.
+ Also you have to remove the dev slot, and give it a separate Web App. 
+ */
+resource appServicePlan 'Microsoft.Web/serverfarms@2022-09-01' = {
+  name: '${appName}-plan'
+  location: location
+  sku: {
+    name: 'S1'
+    tier: 'Standard'
+  }
+  properties: {
+    reserved: true
+  }
+}
+
+resource voteDB 'Microsoft.DocumentDB/databaseAccounts@2023-04-15' = {
+  name: '${appName}-db'
+  location: location
+  tags: {
+    defaultExperience: 'Azure Cosmos DB for MongoDB API'
+    'hidden-cosmos-mmspecial': ''
+  }
+  kind: 'MongoDB'
+  identity: {
+    type: 'None'
+  }
+  properties: {
+    enableFreeTier: false
```

Comment:
Would 1000 RCU/s suffice? (free tier)
https://learn.microsoft.com/en-us/azure/cosmos-db/free-tier

Reply:
Is it 1000 RU/s in total or per second? If it's per second i guess it would work

Reply:
Seems like 1000RCs (operations ish?) per second can be free, but peaks will cost additionally, as per the docs' examples.

We might be able to provision it such that only 1000RC/s can be used at one time. Latency is not super critical during the voting IMO, it takes time registering all votes anyway

Reply:
I think I have accomplished free tier with a 1000RU/s limit now. My only concern is if there is thrown an error if rate limiting occurs. If I understand the docs correctly "DisableRateLimitingResponses" will fix this by letting the backend wait and retry the operation.

https://learn.microsoft.com/en-us/azure/cosmos-db/mongodb/prevent-rate-limiting-errors

Reply:
When testing the free tier with rate limiting I got this strange error. It works on dev but not in prod. Cannot create a new collection, maybe the RU/s are distributed across the collections, but that does not make sense either... 🤔 

https://github.com/NTNUI/vote2/assets/41023500/242a716d-b9bd-4208-99ca-625b39bb855d

Reply:
Might be due to opptak prod which competes for the 1000 free RU/s?
Let's go with the standard tier and check costs every now and then